docs/antrea-gke.md

```diff
@@ -0,0 +1,128 @@
+# Deploying Antrea on a GKE cluster
+
+We support running Antrea inside of GKE clusters on Ubuntu host. Antrea would operate
+in noEncap mode, in which no encapsulation is required for any kind of traffic (Intra node,
+Inter node etc) and enforcing network policies using OVS.
+
+For details on Antrea noEncap Mode, refer to this. [TODO]
+
+## GKE Prerequisites
+
+1. Install the Google Cloud SDK (gcloud). Refer to this (https://cloud.google.com/sdk/install)
+
+    ```bash
+    curl https://sdk.cloud.google.com | bash
+    ```
+
+2. Make sure you are authenticated to use the Google Cloud API
+
+    ```bash
+    export ADMIN_USER=user@email.com
+    gcloud auth login
+    ```
+
+3. Create a project or use an existing one
+
+    ```bash
+    export GKE_PROJECT=gke-clusters
+    gcloud projects create $GKE_PROJECT
+    ```
+
+## Creating the cluster
+
+You can use any method to create a GKE cluster (gcloud SDK, gcloud Console etc). The example
+given here is using the Google Cloud SDK.
+
+Note: Antrea is supported on Ubuntu hosts only for GKE cluster. Also, its must to select service
```

Comment:
**Note**:
better to emphasize on the note with **

Reply:
Done